packages/page-calendar/src/DayItem.tsx

```diff
@@ -154,13 +157,18 @@ function DayItem ({ className, item: { date, info, type } }: Props): React.React
     [info, t, type]
   );
 
+  const exportCalClassNames = showAllEvents ? 'exportCal exportCal-allEvents' : 'exportCal';
+
   return (
     <div className={className}>
+      {showAllEvents &&
+        <div className='itemDate'>{date.toLocaleString('en-EN', options)}</div>
```

Comment:
Will need to check the rest, it could be that we swap to non-locale formats. But generally would not want to enforce the `en` locale.